e2etest/bgp_test.go

```diff
@@ -72,30 +72,28 @@ type containerConfig struct {
 var _ = ginkgo.Describe("BGP", func() {
 	var cs clientset.Interface
 	var f *framework.Framework
-	containersConf := []containerConfig{
-		{
-			name: frrIBGP,
-			nc: frrconfig.NeighborConfig{
-				ASN:      IBGPAsn,
-				Password: "ibgp-test",
-			},
-			rc: frrconfig.RouterConfig{
-				ASN:      IBGPAsn,
-				BGPPort:  179,
-				Password: "ibgp-test",
-			},
+	ebgpContainerConfig := containerConfig{
+		name: frrIBGP,
+		nc: frrconfig.NeighborConfig{
+			ASN:      IBGPAsn,
+			Password: "ibgp-test",
```

Comment:
name frrIBGP ? and ASN need to different maybe we should just call them ASN1 and ASN2 ?

Reply:
This is the very same configuration we had in place before, I just cut the slice in two elements. Apart from naming the elements, I think everything else does not belong to this PR (or at least to this commit).